packages/@aws-cdk/aws-servicecatalog/lib/tag-options.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Defines a Tag Option, which are similar to tags
+ * but have multiple values per key.
+ */
+export class TagOptions {
+  /**
+  * List of CfnTagOption
+  */
+  public readonly tagOptionsMap: {[key: string]: string[]};
+
+  constructor(tagOptionsMap: {[key: string]: string[]}) {
+    this.tagOptionsMap = tagOptionsMap;
```

Comment:
Like I wrote before, I think this should save a defensive copy of the passed map:

```suggestion
    this.tagOptionsMap = { ...tagOptionsMap };
```

So that mutating the `tagOptionsMap` after creating `TagOptions` does not cause any changes in the instance.